samples/highcharts/demo/music-race-chart/demo.js

```diff
@@ -0,0 +1,196 @@
+const btn = document.getElementById('play-pause-button'),
+    input = document.getElementById('play-range'),
+    startYear = 1973,
+    endYear = 2021;
+
+// General helper functions
+const getRevenue = (record, year) => {
+    const revenue = record[year];
+    if (typeof revenue !== 'undefined') {
+        return revenue;
+    }
+    return 0;
```

Comment:
If `record[year]` is either a number or undefined, this could be shortened to 

`const getRevenue = (record, year) => record[year] || 0`

In that case, maybe we don't need a helper function for it either?